input/pagecontent/export.md

```diff
@@ -463,7 +477,7 @@ Required Fields:
       <td>An array of deleted file items following the same structure as the <code>output</code> array.
       <br/>
       <br/>
-        When a <code>_since</code> timestamp is supplied in the export request, this array SHALL be populated with output files containing FHIR Transaction Bundles that indicate which FHIR resources would have been returned, but have been deleted subsequent to that date. If no resources have been deleted or the <code>_since</code> parameter was not supplied, the server MAY omit this key or MAY return an empty array.
+        The ability to convey deleted resources is important in cases when a server may have previously exported data and wishes to indicate that these data should be removed from downstream systems. When a <code>_since</code> timestamp is supplied in the export request, this array SHOULD be populated with output files containing FHIR Transaction Bundles that indicate which FHIR resources would have been returned, but have been deleted subsequent to that date. If no resources have been deleted, or the <code>_since</code> parameter was not supplied, or the server has other reasons to avoid exposing these data, the server MAY omit this key or MAY return an empty array. Resources that appear in the 'deleted' section of an export manifest SHALL NOT appear in the 'output' section of the manifest.
```

Comment:
>Transaction Bundles that indicate which FHIR resources would have been returned, but have been deleted subsequent to that date

I'm being pedantic here, but... imagine if resource X hasn't been deleted, and also hasn't changed since the `_since` time. Then it wouldn't have been returned in this $export invocation, and so the "would have been returned" logic isn't triggered -- so maybe we should say something like:

>Transaction Bundles that indicate which FHIR resources match the `$export` criteria, but have been deleted subsequent to that date